RELEASE.md

```
@@ -10,39 +10,53 @@ You can manually trigger the release workflow from the [GitHub Actions UI](https
+
+2. Triggering the workflow with following requied inputs:
+
+   - **Stable release**
+     - `release_branch_name`: e.g., `release/xrpl@4.3.8` or `release-xrpl-4.3.8`  (must start with `release-`/`release/`, **case-insensitive**).
+     - `package_name`: e.g., `xrpl`.
+     - `npmjs_dist_tag`: `latest`.
+
+  Example: `release_branch_name=release/xrpl@4.3.8`, `package_name=xrpl`, `npmjs_dist_tag=latest`.
 
-➡️ Example:
+   - **Beta release** (publishes as `<tag>-experimental`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)